python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -0,0 +1,332 @@
+                  gdf_datatype=gdf_datatype)
+                
+    
+    @staticmethod
+    def _fit(model, X_df, y_df, r): 
+        return model.fit(X_df, y_df)
+    
+    @staticmethod
+    def _predict(model, X, r): 
+        return model._predictGetAll(X)
+    
+    def fit(self, X, y):
+        """
+        Fit the input data to Random Forest classifier
+
+        Parameters
+        ----------
+        X : Acceptable format: dask-cudf. Dense matrix (floats or doubles) of shape (n_samples, n_features).
+        y : Acceptable format: dask-cudf. Dense matrix (floats or doubles) of shape (n_samples, 1)
+        """
+        c = default_client()
+
+        X_futures = c.sync(_extract_ddf_partitions, X)
+        y_futures = c.sync(_extract_ddf_partitions, y)
+                                       
+        f = list()
+        for w, xc in X_futures.items():     
+            f.append(c.submit(RandomForestClassifier._fit, self.rfs[w], xc, y_futures[w], random.random(),
+                             workers=[w]))            
+                               
+        wait(f)
+        
+        return self
+    
+    def predict(self, X):              
+        """
+        Predicts the labels for X.
+
+        Parameters
+        ----------
+        X : Acceptable format: dask-cudf. Dense matrix (floats or doubles) of shape (n_samples, n_features).
+
+        Returns
+        ----------
+        y: NumPy
+           Dense vector (int) of shape (n_samples, 1)
+
+        """  
+        c = default_client()
+        workers = c.has_what().keys()
+        
+        ws = list(zip(workers, list(range(len(workers)))))
+
+        X_Scattered = c.scatter(X)   
+                
+        f = list()
+        for w, n in ws:
+            f.append(c.submit(RandomForestClassifier._predict, self.rfs[parse_host_port(w)], X_Scattered, random.random(),
+                             workers=[w]))
+                    
+        wait(f)
+
+        indexes = list()
+        rslts = list()
+        for d in range(len(f)):   
+            rslts.append(f[d].result())
+            indexes.append(0)
+                    
+        pred = list()
+                
+        for i in range(len(X)):
+            classes = dict()
+            max_class = -1
+            max_val = 0
+            
+            for d in range(len(rslts)):               
+                for j in range(self.n_estimators_per_worker[d]):
+                    sub_ind = indexes[d] + j
+                    cls = rslts[d][sub_ind]
+                    if cls not in classes.keys():
+                        classes[cls] = 1
+                    else:
+                        classes[cls] = classes[cls] + 1
+
+                    if classes[cls] > max_val:
+                        max_val = classes[cls]
+                        max_class = cls
+
+                indexes[d] = indexes[d] + self.n_estimators_per_worker[d]
+
+            pred.append(max_class)
+            
```

Comment:
```suggestion
```

Too many white lines, did flake8 not whine about it?

Reply:
Didn't notice anything.